app/container/controller.js

```diff
@@ -25,6 +25,16 @@ export default Controller.extend({
 
   }),
 
+  containers: computed('model.containers.[]', function() {
+    return (get(this, 'model.containers') || []).map((container) => {
+      set(container, 'type', 'container');
```

Comment:
This seems like a very strange hack..  You take the model (which is a pod `A` that has a container `B`) and modifying the pod in the store by adding a copy of A to B, then add a containers array to the pod in the container that has just the one container name.

It seems like you're trying to work around pod.containers[] not getting typified into container models correctly because the type is `/v3/project/schema/container` and there's no good way to correlate that.  

I talked to @ibuildthecloud about fixing that in Norman so it will return `/v3/project/<id>/schema/container` instead of `/v3/project/schema/container`.  That would make it correlate to the self link of the schema, which _typeify can then support.

In the meantime if you want to fix the deserialization, do it in the pod model's mangleIn and just set the type of all the containers to `container`.